### PR TITLE
Change dir

### PR DESCRIPTION
**Pull request by khalford**
Changed the directory structure as Pylint and Python are not importing the same as each other and one will bug out.

### CODE REVIEW

File: cloud_chatops/lib/get_github_prs.py
Comment (meoflynn) [inline cloud_chatops/lib/get_github_prs.py:1]: Do we need to add any unit tests for this module?

**Review by meoflynn — CHANGES_REQUESTED** [review]
LGTM, I just had a question about unit tests

File: cloud_chatops/lib/get_github_prs.py
Comment (khalford) [reply cloud_chatops/lib/get_github_prs.py:1]: Yes there will need to be unit tests. However, I will write them in a future PR

File: cloud_chatops/lib/get_github_prs.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        :return: A dictionary of repos and their PRs.
+        """
+        responses = []
+        for repo in self.repos:
+            url = f"https://api.github.com/repos/{self.owner}/{repo}/pulls"
+            responses += self._http_handler.make_request(url)
+        return responses
+
+    @staticmethod
+    def _responses_to_dataclasses(responses: List[Dict]) -> List[PrData]:
```
Comment (anish-mudaraddi) [inline cloud_chatops/lib/get_github_prs.py:51]: missing docstring

Comment (anish-mudaraddi) [reply cloud_chatops/lib/get_github_prs.py:51]: function could be named better - like parse_responses or build_responses

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]
Looks like you've got a lot of changes shared with this PR - https://github.com/stfc/SCD-OpenStack-Utils/pull/142

I would recommend closing this one - getting #142 merged and re-opening this one as it will make reviewing easier for us

File: cloud_chatops/lib/get_github_prs.py (hunk 1 above)
Comment (khalford) [reply cloud_chatops/lib/get_github_prs.py:51]: 👍

**Review by meoflynn — DISMISSED** [review]

File: cloud_chatops/lib/pr_reminder.py [hunk 2]
```diff
@@ -16,20 +16,24 @@ class PostPRsToSlack:
     """
     This class handles the Slack posting.
     """
-    # The default channel is dev-chatops
-    def __init__(self, mention=False, channel="C06U37Y02R4"):
-        self.channel = channel
+
+    def __init__(self, mention=False):
+        self.channel = "C06U37Y02R4"  # STFC-cloud: dev-chatops
```
Comment (anish-mudaraddi) [inline cloud_chatops/lib/pr_reminder.py:21]: why did you change this? - seems like it would be useful to change channel rather than hardcoding it. 
If you feel differently - perhaps setting a constant variable at the top of the file and referencing it here would be better?

File: cloud_chatops/lib/pr_reminder.py [hunk 3]
```diff
@@ -194,7 +199,9 @@ def _check_pr_age(time_created: str) -> bool:
         opened_date = datetime.fromisoformat(time_created).replace(tzinfo=None)
         datetime_now = datetime.now().replace(tzinfo=None)
         time_cutoff = datetime_now - timedelta(days=30 * 6)
-        return opened_date < time_cutoff
+        if opened_date < time_cutoff:
+            return True
+        return False
```
Comment (anish-mudaraddi) [inline cloud_chatops/lib/pr_reminder.py:204]:
```suggestion
        return opened_date < time_cutoff
```

**Review by anish-mudaraddi — CHANGES_REQUESTED** [review]
looks good - just a few nitpicks

File: cloud_chatops/lib/pr_reminder.py [hunk 4]
```diff
@@ -179,7 +181,10 @@ def _github_to_slack_username(self, user: str) -> str:
         :return: Slack ID or GitHub username
         """
         if user not in self.slack_ids:
-            user = self.default_user_id
+            # If the PR author is not in the Slack ID mapping
+            # then we set the user to mention as David Fairbrother
+            # as the team lead to deal with this PR.
+            user = "U01JG0LKU3W"
```
Comment (anish-mudaraddi) [inline cloud_chatops/lib/pr_reminder.py:187]: Also you've hardcoded this here too - you could again set it as a constant variable at the top of the file 
Or better yet in a .conf file that the program reads in - left for a future PR?

File: cloud_chatops/lib/pr_reminder.py (hunk 2 above)
Comment (khalford) [reply cloud_chatops/lib/pr_reminder.py:21]: That is the default channel. Channel is taken on line 29 in the run function. I will set the variable how you mentioned

File: cloud_chatops/lib/pr_reminder.py (hunk 4 above)
Comment (khalford) [reply cloud_chatops/lib/pr_reminder.py:187]: Yes that's a good idea

File: cloud_chatops/lib/pr_reminder.py (hunk 3 above)
Comment (khalford) [reply cloud_chatops/lib/pr_reminder.py:204]: 👍

**Review by anish-mudaraddi — APPROVED** [review]
